lib/x11regressiontest.pm

```diff
@@ -454,9 +454,13 @@ sub post_fail_hook() {
 }
 
 sub start_firefox() {
+    my ($chown_ser_dev) = shift;
     mouse_hide(1);
 
     x11_start_program 'xterm';
+    if ($chown_ser_dev) {
+        testapi::assert_script_sudo("chown $testapi::username /dev/$testapi::serialdev");
+    }
```

Comment:
the `chown` on the serialdev should probably be called by scheduling `tests/x11regressions/x11regressions_setup.pm` unconditionally.